es/02-javascript/02-practica/start-here/spec/entities.js

```diff
@@ -84,7 +84,7 @@ describe('Entities library', function () {
       expect(character.mp).toBe(0);
       expect(character.hp).toBe(0);
       expect(character.maxMp).toBe(0);
-      expect(character.maxHp).toBe(15);
```

Comment:
This is not an error, default characters are assigned a minimum `maxHp` for they to be able to resurrect.